### DISCUSSION TITLE
Ubuntu repository problem

### DISCUSSION THREAD

**Discussion by username-mendoza** [roxy-wi/roxy-wi#397] [General]
Goodmorning,
Here:

![Screenshot from 2024-09-28 10-32-29](https://github.com/user-attachments/assets/49ca22cb-e253-4e1c-b208-78b6b2e255c7)

![Screenshot from 2024-09-28 10-33-58](https://github.com/user-attachments/assets/87e45c91-731c-42f4-908e-a52e77400baf)

**Comment by Aidaho12** [comment]
Hello, there is no version for noble eat. Will added in the next release
